handlers/auth_test.go

```diff
@@ -139,30 +139,58 @@ func TestCreateConnectionCode(t *testing.T) {
 }
 
 func TestGetConnectionCode(t *testing.T) {
-	mockDb := mocks.NewDatabase(t)
-	aHandler := NewAuthHandler(mockDb)
+	teardownSuite := SetupSuite(t)
+	defer teardownSuite(t)
+	aHandler := NewAuthHandler(db.TestDB)
 
 	t.Run("should return connection code from db", func(t *testing.T) {
-		creationDate, _ := time.Parse(time.RFC3339, "2000-01-01T00:00:00Z")
-		existingConnectionCode := db.ConnectionCodesShort{
-			ConnectionString: "test",
-			DateCreated:      &creationDate,
+
+		rr := httptest.NewRecorder()
+		handler := http.HandlerFunc(aHandler.GetConnectionCode)
+
+		codeStrArr := []string{"sampleCode1"}
+
+		codeArr := []db.ConnectionCodes{}
+		now := time.Now()
+
+		for i, code := range codeStrArr {
+			code := db.ConnectionCodes{
+				ID:               uint(i),
+				ConnectionString: code,
+				IsUsed:           false,
+				DateCreated:      &now,
+			}
+
+			codeArr = append(codeArr, code)
 		}
-		mockDb.On("GetConnectionCode").Return(existingConnectionCode).Once()
+
+		// Ensure codeArr has at least one element
+		codeShort := db.ConnectionCodesShort{
+			ConnectionString: codeArr[0].ConnectionString,
+			DateCreated:      codeArr[0].DateCreated,
+		}
+
+		db.TestDB.CreateConnectionCode(codeArr)
+
 		req, err := http.NewRequest("GET", "/connectioncodes", nil)
 		if err != nil {
 			t.Fatal(err)
 		}
-		rr := httptest.NewRecorder()
-		handler := http.HandlerFunc(aHandler.GetConnectionCode)
+
+		fetchedCodes := db.TestDB.GetConnectionCode()
 
 		handler.ServeHTTP(rr, req)
 
 		assert.Equal(t, http.StatusOK, rr.Code)
-		expected := `{"connection_string":"test","date_created":"2000-01-01T00:00:00Z"}`
-		assert.EqualValues(t, expected, strings.TrimRight(rr.Body.String(), "\n"))
-	})
+		assert.EqualValues(t, codeShort.ConnectionString, fetchedCodes.ConnectionString)
+		tolerance := time.Millisecond
```

Comment:
@MahtabBukhari Why these extra lines of code?

Reply:
@elraphty I have removed this part of created due which tests are going to be fail All other data is coming same issue is that when we create connection code we pass date time but when it return it have that part in last of date you can check it 

![image](https://github.com/stakwork/sphinx-tribes/assets/98140673/5485babe-a6c7-4831-ac03-e5a0d8d9e224)